wiki/jvm-langs/using-libgdx-with-kotlin.md

```diff
@@ -41,6 +41,8 @@ This guide describes how to migrate an existing libGDX project to Kotlin. You ca
 
 ## Configure Gradle
 
+UPDATE: Gdx-Liftoff has an option to enable Kotlin support in new projects and will handle most of the configuration for you. These instructions remain for posterity.
```

Comment:
If you want to make this stand out visually, you can add `{: .notice--warning}` to the line below. See [here](https://mmistakes.github.io/minimal-mistakes/post%20formats/post-notice/) for more information.